plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteConnectorTest.java

```diff
@@ -84,6 +84,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_NEGATIVE_DATE:
                 return false;
 
+            case SUPPORTS_PREDICATE_EXPRESSION_PUSHDOWN_WITH_LIKE:
             case SUPPORTS_AGGREGATION_PUSHDOWN_COUNT_DISTINCT:
             case SUPPORTS_DROP_COLUMN:
```

Comment:
No requested change. It would be better to move `SUPPORTS_DROP_COLUMN` to before  `SUPPORTS_ADD_COLUMN`.

Reply:
Updated

Reply:
Please separate a commit. It's unrelated to support for `LIKE` pushdown.

Reply:
https://github.com/trinodb/trino/pull/16425 created